plugin/pkg/admission/podtolerationrestriction/admission.go

```diff
@@ -245,20 +239,20 @@ func (p *podTolerationsPlugin) getNamespaceDefaultTolerations(nsName string) ([]
 	if err != nil {
 		return nil, err
 	}
-	return extractNSTolerations(ns, NSDefaultTolerations)
+	return extractNSTolerations(ns, algorithm.AnnotationDefaultTolerations)
```

Comment:
Same comment, we need to accept the alpha annotation for a small amount of time, right?